documentation/modules/exploit/windows/misc/delta_electronics_infrasuite_deserialization.md

```diff
@@ -0,0 +1,67 @@
+## Vulnerable Application
+
+Delta Electronics InfraSuite Device Master versions below `v1.0.5` are vulnerable to
+an unauthenticated .NET deserialization vulnerability. The `Device-Gateway-Status` process
+listens on port `10100` and communicates over UDP. Its `ParseUDPPacket()` method reads user-controlled
+data and passes what it determines to be the packet header to the `BinaryFormatter.Deserialize()` method,
+resulting in code execution as the user running the `Device-Gateway-Status` process.
+
+### Installation Instructions
+
+1. Unzip and run the installer
```

Comment:
Please provide a link

Reply:
Uploaded a copy to https://archive.org/details/infra-suite-device-master-00.00.01ax-64 so should be able to add this as a link in here.

Reply:
Shoot, I didn't even think of that. Thanks, Grant!

Reply:
No problem :)

Reply:
This was fixed in recent commits.